### PR TITLE
Modify EditCommand to display edited person after edit

### LINKED ISSUE #161: Refine edit command result to use view instead

### PR DESCRIPTION
**Pull request by deeyonn**
Resolves #161 and #162

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -104,6 +105,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         model.setPerson(personToEdit, editedPerson);
         model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
+        model.updateViewedPersonList(new SamePersonPredicate(editedPerson));
         return new CommandResult(String.format(MESSAGE_EDIT_PERSON_SUCCESS, editedPerson));
```
Comment (maryjess) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:109]: I think it is better to remove String.format here?

**Review by maryjess — CHANGES_REQUESTED** [review]
Can you also help to remove String.format in `EditCommandTest.java`, lines 43, 64, 77, 93? Thanks!

**Review by JasonCP14 — APPROVED** [review]
As Jesi said, remove the String format, but other than that, LGTM

File: src/test/java/seedu/address/logic/commands/EditCommandTest.java [hunk 2]
```diff
@@ -40,7 +40,7 @@ public void execute_allFieldsSpecifiedUnfilteredList_success() {
         EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder(editedPerson).build();
         EditCommand editCommand = new EditCommand(INDEX_FIRST_PERSON, descriptor);
 
-        String expectedMessage = String.format(EditCommand.MESSAGE_EDIT_PERSON_SUCCESS, editedPerson);
+        String expectedMessage = EditCommand.MESSAGE_EDIT_PERSON_SUCCESS + editedPerson;
```
Comment (cliftonfelix) [inline src/test/java/seedu/address/logic/commands/EditCommandTest.java:43]: I guess what Jessica means is that the output should just be Person edited, and hence you don't need the String.format and editedPerson

**Review by lulucopter — CHANGES_REQUESTED** [review]
Treating a person object as a String might be misleading. Should we create an `editedPersonString = editedPerson.toString()` instead?

File: src/main/java/seedu/address/logic/commands/EditCommand.java (hunk 1 above)
Comment (deeyonn) [reply src/main/java/seedu/address/logic/commands/EditCommand.java]: Line changed to `return new CommandResult(editedPerson.toString());`

File: src/test/java/seedu/address/logic/commands/EditCommandTest.java (hunk 2 above)
Comment (deeyonn) [reply src/test/java/seedu/address/logic/commands/EditCommandTest.java:43]: Line changed to `String expectedMessage = EditCommand.MESSAGE_EDIT_PERSON_SUCCESS;`

**Review by cliftonfelix — APPROVED** [review]
LGTM

**Pull request merged**